src/caselawclient/Client.py

```diff
@@ -753,6 +771,19 @@ def get_properties_for_search_results(self, judgment_uris: list[str]) -> str:
         response = self._send_to_eval(vars, "get_properties_for_search_results.xqy")
         return decode_multipart(response)
 
+    def search_and_parse_results(
+        self, search_parameters: SearchParameters
+    ) -> SearchResults:
+        response = self.advanced_search(search_parameters)
+        response_string = decode_multipart(response)
+        return SearchResults(response_string)
+
+    def search_judgments_and_parse_results(
+        self, search_parameters: SearchParameters
+    ) -> SearchResults:
+        search_parameters.collections = [JUDGMENT_COLLECTION_URI]
```

Comment:
`SearchParameters.collections` defaults to `None`
but here we override it to be `[JUDGMENT_COLLECTION_URI]` so that when this function is called we only return docs in the `JUDGMENT_COLLECTION_URI` collection (even if someone had specified the collections property in the `SearchParameters` input object.